app/src/main/java/com/door43/translationstudio/App.java

```diff
@@ -797,6 +797,35 @@ public static void clearTargetTranslationSettings(String targetTranslationId) {
     }
 
     /**
+     * moves all settings for a target translation to new target translation (such as when target language changed)
+     * @param targetTranslationId
+     */
+    public static void moveTargetTranslationSettings(String targetTranslationId, String newTargetTranslationId) {
```

Comment:
@neutrinog - OK, but that seems like a kludge since it doesn't seem like migration but moving the meta-data saved in app, not the manifest.